infra-as-code/bicep/modules/policy/assignments/README.md

```diff
@@ -44,43 +44,62 @@ The module does not generate any outputs.
 In this example, the `Deny-PublicIP` custom policy definition will be deployed/assigned to the `alz-landingzones` management group.
 
 #### Azure CLI - Deny
+**NOTE: As there is some PowerShell code within the CLI, there is a requirement to execute the deployments in a cross-platform terminal which has PowerShell installed.**
 
 ```bash
 # For Azure global regions
-az deployment mg create \
-  --template-file infra-as-code/bicep/modules/policy/assignments/policyAssignmentManagementGroup.bicep \
-  --parameters @infra-as-code/bicep/modules/policy/assignments/parameters/policyAssignmentManagementGroup.deny.parameters.all.json \
-  --location eastus \
-  --management-group-id alz-landingzones
+
+dateYMD=$(date +%Y%m%dT%H%M%S%NZ)
+NAME="alz-alz-PolicyDenyAssignmentsDeployment-${dateYMD}"
+
+PARAMETERS="@infra-as-code/bicep/modules/policy/assignments/parameters/policyAssignmentManagementGroup.deny.parameters.all.json"
+LOCATION="eastus"
+MGID="alz-landingzones"
+TEMPLATEFILE="infra-as-code/bicep/modules/policy/assignments/policyAssignmentManagementGroup.bicep"
```

Comment:
Can we standardize the order of these so at least templatefile is then followed by parameters?

Reply:
done